ARK Server Manager/ServerSettingsControl.xaml.cs

```diff
@@ -502,5 +502,21 @@ private void MaxXPDino_Reset(object sender, RoutedEventArgs e)
 
             this.Settings.ResetOverrideMaxExperiencePointsDino();
         }
+
+        private void ChatAndNotification_Reset(object sender, RoutedEventArgs e)
+        {
+            if (MessageBox.Show("Click 'Yes' to confirm you want to reset the Chat and Notification section.", "Confirm Reset Action", MessageBoxButton.YesNo, MessageBoxImage.Question) != MessageBoxResult.Yes)
```

Comment:
This is now common code.  We should refactor this to use a common string and parameters.  If you switch to commands, the section name can be a parameter to the command, defined in the XAML, I believe.  Then we don't even have to have multiple commands, just a single one that takes a parameter or two.